client/highlevel-rest/src/test/java/org/elasticsearch/client/HLClientSearchIT.java

```diff
@@ -39,34 +53,90 @@ public void init() {
     }
 
     public void createTestDoc() throws IOException {
+        XContentBuilder mappingsBuilder = randomXContentBuilder();
+        mappingsBuilder.startObject();
+        mappingsBuilder.startObject("mappings");
+        mappingsBuilder.startObject("type");
+        mappingsBuilder.startObject("properties");
+        mappingsBuilder.startObject("title");
+        mappingsBuilder.field("type", "text");
+        mappingsBuilder.field("store", "true");
+        mappingsBuilder.endObject();
+        mappingsBuilder.startObject("content");
+        mappingsBuilder.field("type", "text");
+        mappingsBuilder.field("store", "true");
+        mappingsBuilder.endObject();
+        mappingsBuilder.endObject();
+        mappingsBuilder.endObject();
+        mappingsBuilder.endObject();
+        mappingsBuilder.endObject();
```

Comment:
Anything particular in mind? Or should I leave it for now and deal with it as soon as we need it more often?